book/makefile

```diff
@@ -134,15 +135,11 @@ low-res-book-epub/OEBPS/%.jpg: %.jpg
 	mkdir -p $(dir $@)
 	$(CONVERT_PIC) $< $(REDUCE_PIC_COLOR) $@
 
-low-res-book-epub/OEBPS/%.png: %.png
-	mkdir -p $(dir $@)
-	$(CONVERT_PIC) $< $(REDUCE_PIC_COLOR) $@
```

Comment:
Option 2) you copy/rsync png here instead of convert.  Might like option 1 better...